### DISCUSSION TITLE
Meeting next Tuesday

### DISCUSSION THREAD

**Discussion by saulshanabrook** [saulshanabrook/jupyterlab-accessibility#8] [Meetings]
Hey all,

I was chatting with Quinten and Sara about doing a **meeting Tuesday, maybe at 5:30pm EST,** night if we wanted to collectively make a contribution to JupyterLab.

The goal could be for all of us to fix this issue: https://github.com/jupyterlab/jupyterlab/issues/9874, which is basically a find and replace.

Before the meeting, it would be great if everyone could get set up to develop on JupyterLab, by:

1. Getting your computer set up by install [Miniconda](https://docs.conda.io/en/latest/miniconda.html). If you have windows it's a little more involved, so if you can get on Linux and Mac that is easiest. But can still make it work on Windows. 
2. Find an editor you like. @rodylipson mentioned [Visual Studio Code](https://code.visualstudio.com/), which is what I would recommend as well. You can just use a plain text editor and the command line if you want, whatever you are comfortable with.

If you have any questions on getting set up, I am happy to have a call with you before hand and just make sure things are working properly.

Then during the meeting we could all download the JupyterLab code to your computer, edit it, see the changes reflected, and then "push" your code back to GitHub so that the community can review your change. Of course, in the end, only one of you can actually get the change in, but we can all practice doing it, then just choose one at random to be "merged" (which means accepting your change).

Then, on Wednesday, we have our call again with the larger group, but this time focused just on new contributors.

How does this sound?

**Comment by kali-ransom** [comment]
Next Tuesday works for me Saul! I'll actually prep before this one lol. One thing: my personal laptops are still dead, is Miniconda something that could work on a *very shitty* google tablet?

**Reply by saulshanabrook** [reply]
Ah good question...  probably not so much. But I think we could use some free cloud computing. I'll do a demo of that on Tuesday, on Tuesday, for anyone who couldn't/didn't want to do things locally.

**Reply by saulshanabrook** [reply]
This is the link to try it out: https://gitpod.io/#https://github.com/saulshanabrook/jupyterlab but I can do more of a walkthrough on Tuesday.

**Comment by RodyLipson** [comment]
hey works for me!

**Comment by kali-ransom** [comment]
Thanks Saul!

**Comment by saulshanabrook** [comment]
Another option, instead of getting your computer set up, is to use GitHub code spaces. https://github.com/features/codespaces

It's currently in beta, so you have to request early access. You can try to do that and see when they let you in.
